### ISSUE TITLE
px to pt conversion issue

### ISSUE THREAD

**Issue by nazjunaid** [moroshko/accessible-colors#76]
According to wcag for large text bold you need to be at least 14pt = 18.5px

https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html#visual-audio-contrast-contrast-73-head

This tool defines 18px as a success due to way it rounds however google lighthouse/axe marks it a fail since 13.5pt isn't 14pt

WCAG's own documentation states

_The ratio between sizes in points and CSS pixels is 1pt = 1.333px, therefore 14pt and 18pt are equivalent to approximately 18.5px and 24px._

So based on this I don't think the tool should round 13.5pt to 14pt since it's a WCAG fail.
